gips/XXXX-optimizing-query-fees-sent-to-indexers.md

```diff
@@ -0,0 +1,64 @@
+---
+GIP: "XXXX"
+Title: Optimizing Query Fees sent to Indexers
+Authors: Howard 
+Created: 2023-11-08
+Updated: 2023-11-08
+Stage: Draft
+Category: Economic Parameters
+---
+
+## Abstract
+
+Today, at least 11% of query fees paid by data consumers do not ever make it to either the Gateway or the Indexers that provide the service to the data consumers. This GIP aims to reduce these transactional costs. This can be done without changing any smart contract code; instead, only two parameters need to be tweaked: removing the fees sent to curators and setting the protocol tax to zero. This will simplify how the protocol operates and potentially save GRT both for developers and Indexers.
```

Comment:
~~Minor nitpick, but it is preferred to capitalize all network roles (Curators does not seem to be capitalized).~~  

Edit: after reading through, it strikes me as a good stylistic choice _not_ to capitalize curators. Great work!